python_tests/pytest_core.py

```diff
@@ -75,6 +90,20 @@ def to_jax(t):
         args = map(to_jax, self.args)
         return SampleInput(*args, *self.kwargs.values())
 
+    def python(self):
+        # Flatten Pytorch Tensors into Python Lists
+        def to_python(t):
+            if isinstance(t, torch.Tensor):
+                return list(t.flatten().cpu().numpy())
+            if isinstance(t, torch.dtype):
+                return _torch_to_python_dtype_map[t]
+            return t
+
+        # Note: We assume arguments have flat hierarchy.
+        # TODO Add support for kwargs
+        args = map(to_python, self.args)
+        return SampleInput(*args, *self.kwargs.values())
```

Comment:
Minor suggestion. This is just personal preference; feel free to ignore: 
```suggestion
        args = [
            list(t.flatten().cpu().numpy()) if isinstance(t, torch.Tensor)
            else _torch_to_python_dtype_map[t] if isinstance(t, torch.dtype)
            else t
            for t in self.args]
        return SampleInput(*args, *self.kwargs.values())
```